WalletWasabi/WabiSabi/Crypto/CredentialRequesting/ZeroCredentialsRequest.cs

```diff
@@ -0,0 +1,27 @@
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Crypto.ZeroKnowledge;
+
+namespace WalletWasabi.WabiSabi.Crypto.CredentialRequesting
+{
+	public class ZeroCredentialsRequest : CredentialsRequest
+	{
+		public ZeroCredentialsRequest(
+			long delta,
+			IEnumerable<CredentialPresentation> presented,
+			IEnumerable<IssuanceRequest> requested,
+			IEnumerable<Proof> proofs)
+			: base(delta, presented, requested, proofs)
```

Comment:
I think this class as well as the RealCredentialRequest are unecessary, they have only a validation in the constructor. Anyway, in this case the constructor could be like: 

```suggestion
		public ZeroCredentialsRequest(
			IEnumerable<IssuanceRequest> requested,
			IEnumerable<Proof> proofs)
			: base(0, Array.Empty<CredentialPresentation>(), requested, proofs)
```

Reply:
In this case the validation `if (!IsNullRequest)` is unnecessary.

Reply:
Done